ProcessLib/HT/HTProcess.cpp

```diff
@@ -160,5 +165,54 @@ void HTProcess::setCoupledTermForTheStaggeredSchemeToLocalAssemblers()
         _local_assemblers, _coupled_solutions);
 }
 
+NumLib::LocalToGlobalIndexMap* HTProcess::getDOFTableForExtrapolatorData(
+    bool& manage_storage) const
```

Comment:
Because manage_storage is used only as output parameter, it would be better to return it too:
```c++
std::tuple<bool, DOF*> f()
{
   if (A) return {false, dof};
   return {true, other_dof};
}
```

Reply:
Changed the return type to std::tuple.